openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -78,6 +78,8 @@ def get_value_for_org(cls, org, name, default=None):
         """
         for configuration in cls.objects.filter(values__contains=org, enabled=True).all():
             course_org_filter = configuration.get_value('course_org_filter', None)
```

Comment:
I think we should default to an empty list here, so we are not adding `None` to a list in the check below.